pkg/network/protocols/grpc/monitor_test.go

```diff
@@ -221,14 +221,13 @@ func testGRPCScenarios(t *testing.T) {
 					Method: http.MethodPost,
 				}: 5,
 			},
-			expectedError: true,
 		},
 		{
-			name: "request with large body (1MB) -> b -> request with large body (1MB) -> b",
+			name: "request with large body (5MB) -> b -> request with large body (5MB) -> b",
```

Comment:
do we know what is the current max body size that we can support? I can understand that it is probably related to the gRPC max size, but maybe it is worth to use some UTs to understand our max size

Reply:
unlimited, that depends on the how the http2 is split into packets
I'll run a UT with 1G